app/views/schemes/show.html.erb

```diff
@@ -10,19 +10,28 @@
 
 <%= render partial: "organisations/headings", locals: { main: @scheme.service_name, sub: nil } %>
 
-<%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, "Locations")) %>
+<% if FeatureToggle.new_locations_table_layout_enabled? %>
```

Comment:
I think you could probably just use the `location_toggle_enabled` right? Or is there a reason this behaviour has to be displayed separately to the deactivating/reactivating locations behaviour?

Reply:
There's maybe also an argument for `location_toggle_enabled` and `scheme_toggle_enabled` being the same as well to be honest but maybe those are a bit more independent

Reply:
I hadn't looked at `location_toggle_enabled`, tbh I was assuming each separate ticket/change in functionality would have a separate feature toggle. But are you saying that all of the current changes to the locations pages should all be under the same feature toggle? If so happy to just use `location_toggle_enabled` :).

Reply:
@natdeanlewissoftwire updated this now to use `location_toggle_enabled` instead. I'm using this to toggle the 2/3 scaling on the `app/views/schemes/show.html.erb` page instead of `scheme_toggle_enabled` because I only want to toggle the 2/3 width on the scheme show page when the locations page gets toggled - i.e. it's directly linked to the locations toggle, and not related to the other work on schemes which falls under the schemes toggle.